pkg/storaged/crypto-keyslots.jsx

```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
     const dlg = dialog_open({
         Title: _("Verify key"),
         Body: (
-            <>
-                <p>{_("Make sure the key hash from the Tang server matches one of the following:")}</p>
-
-                <h2 className="sigkey-heading">{_("SHA256")}</h2>
-                { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha256}</p>) }
-
-                <h2 className="sigkey-heading">{_("SHA1")}</h2>
-                { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha1}</p>) }
+            <TextContent>
+                <Text component={TextVariants.p}>{_("Check the key hash with the Tang server.")}</Text>
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)